error-issue/index.ts

```diff
@@ -27,6 +28,7 @@ const GITHUB_REPO = process.env.GITHUB_REPO || 'ampproject/amphtml';
 const [GITHUB_REPO_OWNER, GITHUB_REPO_NAME] = GITHUB_REPO.split('/');
 const GITHUB_ACCESS_TOKEN = process.env.GITHUB_ACCESS_TOKEN;
 const PROJECT_ID = process.env.PROJECT_ID || 'amp-error-reporting';
+const MIN_FREQUENCY = Number(process.env.MIN_FREQUENCY || 2500);
```

Comment:
Default is 2500 here but `ErrorMonitor` defaults to 5000 in the constructor. I would either remove the default value in the constructor, or make it the same value.

Reply:
Will remove.